### PR TITLE
feat: Add RecommenderXBlock to the translation pipeline

### PR DESCRIPTION
**Pull request by shadinaif**
This PR needs https://github.com/openedx/RecommenderXBlock/pull/65 before it's merged.

Refs: FC-0012 OEP-58

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @shadinaif!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: transifex.yml [hunk 1]
```diff
@@ -157,6 +157,14 @@ git:
...
     translation_files_expression: 'translations/paragon/src/i18n/messages/<lang>.json'
 
+  # RecommenderXBlock
+  - filter_type: dir
+    file_format: PO
+    source_file_extension: po
+    source_language: en
+    source_file_dir: translations/RecommenderXBlock/recommender/translations/en/
+    translation_files_expression: 'translations/RecommenderXBlock/recommender/translations/<lang>/'
+
```
Comment (OmarIthawi) [inline transifex.yml:167]:
```suggestion
  # RecommenderXBlock
  - filter_type: dir
    file_format: PO
    source_file_extension: po
    source_language: en
    source_file_dir: translations/RecommenderXBlock/recommender/conf/locale/en/
    translation_files_expression: 'translations/RecommenderXBlock/recommender/conf/locale/<lang>/'
```

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
Thanks Shadi. Could you please use the _new_ standard `conf/locale` directory?

Also add to the github actions file: https://github.com/openedx/openedx-translations/blob/main/.github/workflows/extract-translation-source-files.yml#L41-L43

File: transifex.yml (hunk 1 above)
Comment (shadinaif) [reply transifex.yml]: Done, thank you @OmarIthawi

**Comment by OmarIthawi** [comment]
@shadinaif please add the entry to the github actions file: https://github.com/openedx/openedx-translations/blob/main/.github/workflows/extract-translation-source-files.yml#L41-L43

**Review by OmarIthawi — APPROVED** [review]

**Review by brian-smith-tcril — APPROVED** [review]
LGTM :shipit:

**Review by brian-smith-tcril — CHANGES_REQUESTED** [review]
I tested this by pushing the branch to my fork and it seems to be failing

https://github.com/brian-smith-tcril/openedx-translations/actions/runs/4503779916/jobs/7927421393

**Comment by shadinaif** [comment]
@brian-smith-tcril it's because JS part is still not merged, I'll ping you again when it's good to go. Thank you!

**Comment by shadinaif** [comment]
@brian-smith-tcril  This one will not fail after merging https://github.com/openedx/RecommenderXBlock/pull/68

**Review by brian-smith-tcril — APPROVED** [review]
[Tested on my fork](https://github.com/brian-smith-tcril/openedx-translations/actions/runs/4661000141/jobs/8249750693) and this is working now!

LGTM :shipit:

**Pull request merged**